tests/sensitivities/test_design_distributions.py

```diff
@@ -209,21 +209,69 @@ def test_draw_values_loguniform():
     assert all(10 <= value <= 100 for value in values)
 
 
+def test_that_discrete_matches_with_expected_probabilities():
+    n_samples = 20
+    outcomes = ["a", "b", "c"]
+    probabilities = "0.2,0.3,0.5"
+    dist_params = [",".join(outcomes), probabilities]
+    status, values = dists.sample_discrete(dist_params, n_samples)
+    assert status
+
+    from collections import Counter
```

Comment:
can this be moved to the import section?

Reply:
It can but should it?

Reply:
Yes it should. We tend to do local imports only when we have some circular imports.